crdb/crdbgorm/gorm.go

```diff
@@ -17,7 +17,7 @@ package crdbgorm
 import (
 	"context"
 	"database/sql"
-	"github.com/cockroachdb/cockroach-go/crdb"
+	"github.com/cockroachdb/cockroach-go/v2/crdb"
```

Comment:
any reasons to upgrade this to v2? we should really only be doing this if the API is compatible with before. v1 should be fine.

Reply:
If I remove the `v2`, then when I run the tests, these lines get added to `go.mod` and `go.sum`. is that bad?

```
diff --git a/go.mod b/go.mod
index 4945e3f..b73b6c9 100644
--- a/go.mod
+++ b/go.mod
@@ -3,6 +3,7 @@ module github.com/cockroachdb/cockroach-go/v2
 go 1.13

 require (
+    github.com/cockroachdb/cockroach-go v2.0.1+incompatible
     github.com/jackc/pgx/v4 v4.6.0
     github.com/jinzhu/gorm v1.9.12
     github.com/jmoiron/sqlx v1.2.0
diff --git a/go.sum b/go.sum
index 3f24f28..931b93a 100644
--- a/go.sum
+++ b/go.sum
@@ -1,5 +1,7 @@
 github.com/cockroachdb/apd v1.1.0 h1:3LFP3629v+1aKXU5Q37mxmRxX/pIu1nijXydLShEq5I=
 github.com/cockroachdb/apd v1.1.0/go.mod h1:8Sl8LxpKi29FqWXR16WEFZRNSz3SoPzUzeMeY4+DwBQ=
+github.com/cockroachdb/cockroach-go v2.0.1+incompatible h1:rkk9T7FViadPOz28xQ68o18jBSpyShru0mayVumxqYA=
+github.com/cockroachdb/cockroach-go v2.0.1+incompatible/go.mod h1:XGLbWH/ujMcbPbhZq52Nv6UrCghb1yGn//133kEsvDk=
 github.com/coreos/go-systemd v0.0.0-20190321100706-95778dfbb74e/go.mod h1:F5haX7vjVVG0kc13fIWeqUViNPyEJxv/OmvnBo0Yme4=
 github.com/coreos/go-systemd v0.0.0-20190719114852-fd7a80b32e1f h1:JOrtw2xFKzlg+cbHpyrpLDmnN1HqhBfnX7WDiW7eG2c=
 github.com/coreos/go-systemd v0.0.0-20190719114852-fd7a80b32e1f/go.mod h1:F5haX7vjVVG0kc13fIWeqUViNPyEJxv/OmvnBo0Yme4=
```

Reply:
change the line `module github.com/cockroachdb/cockroach-go/v2` to `module github.com/cockroachdb/cockroach-go`

Reply:
my concern with that is that v2 was already created in this commit: https://github.com/cockroachdb/cockroach-go/commit/499e2dd981f98ec81e74d5888e06a7b30508edf1

and some people have been using the v2 version already, since the `version-module` branch was published here

it seems like bad things will happen if we make a new release that has a smaller version number than what people are using already. but maybe it's fine?

Reply:
if it's published, then yeah make sense to upgrade to v2.